zetkin/app.zetkin.org
# Issue 552/search dialog

## Description
This PR adds a search dialog to all tabbed layout pages. It also changes the style of the collapse button.

## Screenshots
[Add screenshots here]


## Changes
* Adds a button on all organize pages that opens a search dialog when clicked
* Adds a search dialog
  * With loading and error state
  * Allows the user to search for tasks, campaigns, and people
  * Navigates to the url of the resource when clicking the search result 
* Changes the header collapse feature to
  * Only show that button on pages where the user can not scroll, such as views and the calendar. Other pages where the user can scroll can no longer be collapsed
  * Changes the style of that button

## Notes to reviewer
* Test searching works correctly
* Check that loading state loads properly
* Try and get error state to work properly
* Click each type of search data type (task, campaign person) and check it navigates to the correct page.

## Related issues
Resolves #552
